### PR TITLE
Add ws_data_dir to wheel

### PR DESCRIPTION
**Pull request by danielg55**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,9 @@ COPY marine ./marine
 
 ENV PY="/opt/python/cp38-cp38/bin/python"
 
-RUN mkdir marine/.wslibs && \
+RUN mkdir marine/.ws_data_dir && \
+    rsync -r --exclude "lib*.so*" /build/run/* marine/.ws_data_dir && \
```
Comment (yehonatanz) [inline Dockerfile]: Be consistent with your usage of `rsync` and `cp` please

**Review by yehonatanz — CHANGES_REQUESTED** [review]
I'd like to see a test verifying your changes if that's not too expensive, and of course I assume you have manually tried that and it works

**Comment by yehonatanz** [comment]
Also I think you should modify our `setup.py` and perhaps prefer having a single `.ws` dir with `libs` and `data` inside

**Comment by danielg55** [comment]
I can't think of a way to test this change, since it should only affect plugins.

**Review by yehonatanz — APPROVED** [review]

**Pull request merged**